src/components/login-card/index.tsx

```diff
@@ -32,7 +32,7 @@ const LoginCard = (props: any) => {
     }, []);
     return (
       <Card className="card-login">
-        <p className="login-header">Login</p>
+        <h2>Login</h2>
```

Comment:
Did we have a standard for h and p tags?

Reply:
Oh nvm I see it. Naisu.

Reply:
header and paragraph styling should be handled by the d4sd components thing